### PR TITLE
Port recent DFA framework changes to 2.9.x

### PR DESCRIPTION
**Pull request by mavasani**

### CODE REVIEW

**Comment by dotpaul** [comment]
Guess we'll have to muck with ServicePointManager.SecurityProtocol for now?

File: azure-pipelines.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
   pool:
     name: NetCorePublic-Pool
     queue: buildpool.windows.10.amd64.vs2017.open
-  timeoutInMinutes: 30
+  timeoutInMinutes: 40
```
Comment (mavasani) [inline azure-pipelines.yml:23]:
```suggestion
  timeoutInMinutes: 30
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -49,7 +49,7 @@ jobs:
   pool:
     name: NetCorePublic-Pool
     queue: BuildPool.Windows.10.Amd64.ES.VS2017.Open
-  timeoutInMinutes: 30
+  timeoutInMinutes: 40
```
Comment (mavasani) [inline azure-pipelines.yml:52]: 30

Comment (mavasani) [inline azure-pipelines.yml:52]:
```suggestion
  timeoutInMinutes: 30
```

**Comment by mavasani** [comment]
@dotpaul can you review? Tests are now green!

**Pull request merged**